tools/spi-hdlc-adapter/spi-hdlc-adapter.c

```diff
@@ -102,29 +103,33 @@ static int gMode = MODE_PTY;
 static int gMode = MODE_STDIO;
 #endif
 
-static const char* gSpiDevPath = NULL;
+static const char* gSpiDevPath     = NULL;
```

Comment:
Actually, does the style guide make a differentiation between static global scope and static local scope? If not, it probably should.